### ISSUE TITLE
Viewport fix in 5.1.10 causes the original bug

### ISSUE THREAD

**Issue by jalovatt** [storybookjs/storybook#7646]
Our project hasn't had any trouble up to this point - for us, the fix in #7173  is now causing the problem it's meant to solve. I updated Storybook this morning and got:

![http://imgur.com/9ziWTOVl.png](http://imgur.com/9ziWTOVl.png)

- Reverting the CSS to `height: 100%` puts things back to normal.
- We are _not_ using `addon-backgrounds`, if that helps.
- We don't have much going on for addons that might be conflicting:
  ```javascript
  import 'storybook-addon-interaction/register';
  import '@storybook/addon-knobs/register';
  import '@storybook/addon-viewport/register';
  import 'storybook-addon-intl/register';
  ```
- Our DOM looks like:
  ![http://i.imgur.com/FSQCHBQ.png](http://i.imgur.com/FSQCHBQ.png)
- Seeing this on Firefox 68.0.1 and Chrome 75.0.3770.142, both running on Ubuntu 18.04. My apologies for not having more system information; `npx` throws an error that I haven't been able to resolve:
  ```
  npx -p @storybook-cli@next sb info
  > Error: Invalid package name "@storybook-cli": name can only contain URL-friendly characters
  ```

Cheers.

**Comment by shilman** [comment]
@Autarc can you look into this?

**Comment by Autarc** [comment]
Yeah I'm going to check it the next days!

@jalovatt Didn't face that issue before but sorry for the inconvenience.

**Comment by jalovatt** [comment]
@Autarc Not a problem. Cheers.

**Comment by shilman** [comment]
@Autarc did you have a chance to look at this? I'm going to have to back out your change if you can't get to this. Thanks!

**Comment by Autarc** [comment]
@shilman Unfortunately I didn't had the chance yet so it might be better to revert if a fix which should have helped causes more trouble. Will come back with a more reliable solution then in the future!

**Comment by ndelangen** [comment]
This should be fixed in 5.2-rc.0
